Utilities/changing_names_between_locales.py

```diff
@@ -0,0 +1,179 @@
+""""
+This script contains functions which help in creating TXT file which contains
+proper nouns from TSVs, merging proper nouns TXT files, and to rename names in TSVs
```

Comment:
'merging proper nouns  TXT files' --> makes no sense. What does this phrase convey?